resources/views/components/game/multiline-avatar.blade.php

```diff
@@ -59,10 +73,6 @@ class="w-9 h-9"
                 @if ($consoleName && !$consoleId)
                     <span class="block text-xs tracking-tighter mt-px">{{ $consoleName }}</span>
                 @endif
-
-                @if ($consoleId && !$consoleName)
-                    <span class="block text-xs tracking-tighter mt-px">{{ config('systems')[$consoleId]['name'] }}</span>
-                @endif
```

Comment:
if `consoleId` is set, `consoleName` will be pulled from the model, so the code will always go through the logic on line 54.